src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 200px;
+  height: 408px;
+  border: 1px;
```

Comment:
if you want to use border you have to specify the width and the color of the border, also make sure that the border counts to card's height and width. Use correct `box-sizing`